### PR TITLE
MAT-10341: update config to match other toolchain order; update getStructureDefinitionById to ignore version (if present) for now

### PR DESCRIPTION
**Pull request by nmorasb**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-10341](https://jira.cms.gov/browse/MAT-10341)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/madiefhirservice/config/HapiFhirConfig.java [hunk 1]
```diff
@@ -156,8 +156,8 @@ public IValidationSupport uscore6ValidationSupportChain(@Autowired FhirContext u
             validationConfig);
 
     return new ValidationSupportChain(
-        npmPackageSupport,
         vsesValidationSupport,
+        npmPackageSupport,
```
Comment (adongare) [inline src/main/java/gov/cms/madie/madiefhirservice/config/HapiFhirConfig.java:160]: is this to improve the performance?

**Review by adongare — APPROVED** [review]

File: src/main/java/gov/cms/madie/madiefhirservice/config/HapiFhirConfig.java (hunk 1 above)
Comment (nmorasb) [reply src/main/java/gov/cms/madie/madiefhirservice/config/HapiFhirConfig.java:160]: Not necessarily to improve performance..I just noticed that the other validation support chains had been previously reorganized to invoke vses first, then npm second. So was updating this one to match
